src/code/ucode_disas.c

```diff
@@ -0,0 +1,324 @@
+#include <global.h>
+
+#define F3DZEX_CONST(name) \
+    { name, #name }
+#define F3DZEX_FLAG(set, unset) \
+    { set, #set, #unset }
+#define F3DZEX_RENDERMODE(name, mask) \
+    { #name, name, mask }
+#define F3DZEX_SETRENDERMACRO(name, shift, len, value0, value1, value2, value3) \
+    {                                                                           \
+        name, shift, len, {                                                     \
+            { #value0, value0 }, { #value1, value1 }, { #value2, value2 }, {    \
+#value3, value3                                                 \
```

Comment:
Not sure whether the formatter did that but is there any chance the style here could be improved somehow?